packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -402,17 +406,25 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 	 * Creates an uninitialised device that can be referenced by the timeline and mappings.
 	 * @param deviceId Id used by the mappings to reference the device.
 	 * @param deviceOptions The options used to initalize the device
+	 * @param options Additional options
 	 * @returns A promise that resolves with the created device, or rejects with an error message.
 	 */
 	public async createDevice(
 		deviceId: string,
-		deviceOptions: DeviceOptionsAnyInternal
+		deviceOptions: DeviceOptionsAnyInternal,
+		options?: { signal?: AbortSignal }
```

Comment:
Since `signal` has the type `AbortSignal`, then why not call it `abortSignal`? (You do it in your makeAbortable function)

Reply:
In the public API, I'm Following conventions that Node.js uses https://blog.logrocket.com/complete-guide-abortcontroller-node-js/